hc/api/models.py

```diff
@@ -114,7 +114,8 @@ def to_dict(self):
             "tags": self.tags,
             "timeout": int(self.timeout.total_seconds()),
             "grace": int(self.grace.total_seconds()),
-            "n_pings": self.n_pings
+            "n_pings": self.n_pings,
+            "status": self.status
```

Comment:
Use self.get_status() to compute the correct status on the fly. self.status may be out of date if sendalerts is not running or is swamped with work. Also, should probably include a "in_grace_period" flag: 

    "status": "grace" if self.in_grace_period() else self.get_status()